### PR TITLE
Remove old code, fix translation domains, bump compatibility

### PR DESCRIPTION
**Pull request by Hlavtox**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | See below
| Type?             | bug fix
| BC breaks?        | yes
| Deprecations?     | no
| Fixed ticket?     | Fixes #https://github.com/PrestaShop/PrestaShop/issues/32175
| Sponsor company   | 
| How to test?      | 

### Description
- Dropped universal analytics support. **DO NOT RELEASE THIS MODULE BEFORE 1.7.2023!**
- Dropped legacy translation system and migrated to the new one.
- Fixes translation domain to allow proper translations and export.
- Removes compatibility with 1.6 (this was the last module) and bumps compatibility to 1.7.6 (first version with the new price format system).

### CODE REVIEW

File: config.xml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <module>
 	<name>ps_googleanalytics</name>
 	<displayName><![CDATA[Google Analytics]]></displayName>
-	<version><![CDATA[4.2.2]]></version>
+	<version><![CDATA[4.3.0]]></version>
```
Comment (matks) [inline config.xml:5]: If we drop old compatibility, maybe we should do it in a 5.0.0 ? So people understand it's a big change?

**Review by matks — COMMENTED** [review]
Code is good for me 👍 I'm just not sure if we should do a 4.3.0 or a 5.0.0

**Review by tleon — APPROVED** [review]
I agree with @matks a drop in PS version support should be a major module version

File: config.xml (hunk 1 above)
Comment (kpodemski) [reply config.xml:5]: @Hlavtox it is actually **required** to bump the major version if we change PS compatibility version :)

Comment (kpodemski) [reply config.xml:5]: https://github.com/PrestaShop/ADR/blob/master/0020-module-version-bump-after-changing-min-core-version.md

**Review by M0rgan01 — APPROVED** [review]

**Review by kpodemski — DISMISSED** [review]
Version bump required

File: classes/Hook/HookDisplayBeforeBodyClosingTag.php [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+                                ],
+                            ],
+                        ];
+                        $gaScripts .= $this->module->getTools()->renderEvent(
+                            'remove_from_cart',
+                            $eventData
+                        );
+                    }
+                } elseif (is_array($gacart)) {
+                    $it = new RecursiveIteratorIterator(new RecursiveArrayIterator($gacart));
```
Comment (matks) [inline classes/Hook/HookDisplayBeforeBodyClosingTag.php:116]: 😛

**Review by kpodemski — APPROVED** [review]

**Review by kpodemski — APPROVED** [review]

**Review by matks — APPROVED** [review]

**Comment by Hlavtox** [comment]
@aniszr @hibatallahAouadni Issue fixed, you can test it :-)

**Comment by nicosomb** [comment]
Thank you @Hlavtox !

**Pull request merged**